### PR TITLE
add b,w,e keymappings

### PR DESCRIPTION
**Pull request by r2dev2**

### CODE REVIEW

**Comment by r2dev2** [comment]
I also just added in support for the `a` key which fixes #8.

File: docs-vim.js [hunk 1]
```diff
@@ -8,6 +8,12 @@ vim = {
     }
 };
 
+var multiMaps = {
+    "b": [["ArrowLeft", true]],
+    "e": [["ArrowRight", true]],
+    "w": [["ArrowRight", true], ["ArrowRight", true], ["ArrowLeft", true]]
+};
+
```
Comment (matthewsot) [inline docs-vim.js]: Can you add this as `vim.multiMaps`, e.g., on line 8? Just to keep all the configuration in one spot.

**Review by matthewsot — APPROVED** [review]
Looks great to me, thanks so much for putting this together!

The only thing I would ask/suggest is moving `multiMaps` into the `vim` object, like `vim.keys` currently is.

File: docs-vim.js (hunk 1 above)
Comment (r2dev2) [reply docs-vim.js]: Added in 85b4f01

**Pull request merged**